cmd/kind/create/create.go

```diff
@@ -28,10 +28,9 @@ import (
 // NewCommand returns a new cobra.Command for cluster creation
 func NewCommand() *cobra.Command {
 	cmd := &cobra.Command{
-		// TODO(bentheelder): more detailed usage
 		Use:   "create",
 		Short: "Creates one of [cluster]",
-		Long:  "Creates one of [cluster]",
+		Long:  "Creates one of local kubernetes cluster (cluster)",
```

Comment:
`kubernetes` should be uppercase everywhere in stdout and user docs. [1]

Reply:
Thank you for catching that